src/main/webapp/app/quiz/re-evaluate/short-answer-question/re-evaluate-short-answer-question.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ShortAnswerQuestion } from '../../../entities/short-answer-question';
+
+@Component({
+    selector: 'jhi-re-evaluate-short-answer-question',
+    template: `<jhi-edit-short-answer-question [question]="question"
+                                                [questionIndex]="questionIndex"
+                                                [reEvaluationInProgress]="true"
+                                                (questionUpdated)="questionUpdated.emit()"
+                                                (questionDeleted)="questionDeleted.emit()"
+                                                (questionMoveUp)="questionMoveUp.emit()"
+                                                (questionMoveDown)="questionMoveDown.emit()">
+               </jhi-edit-short-answer-question>`,
+    providers: []
+})
+export class ReEvaluateShortAnswerQuestionComponent {
+    /**
+     question: '=',
```

Comment:
what are these comments for?

Reply:
Deleted the comments as they are not needed